cartography/data/jobs/cleanup/gcp_sql_cleanup.json

```diff
@@ -0,0 +1,29 @@
+{
+    "statements": [
+      {
+        "query": "MATCH (i:GCPSQLInstance)<-[:RESOURCE]-(:GCPProject{accountid: {GCP_PROJECT_ID}}) WHERE i.lastupdated <> {UPDATE_TAG} WITH i LIMIT {LIMIT_SIZE} DETACH DELETE (i) return COUNT(*) as TotalCompleted",
```

Comment:
let's use n for alias instead of i :). To keep it consistent with other cleanup scripts.